index.html

```diff
@@ -86,201 +86,16 @@ <h3>My Recent Works</h3>
                     </div>
                 </div>
             </div>
+            <section class="item1" id="main-project">
+               
+            </section>
+            <ul id="work-projects" class="grid-desktop">
+               
+    
+            </ul>
```

Comment:
- [ ] The pop-up modal in your design on the mobile screen width goes all the way down which is not right. kindly consider applying the `overflow-y: scroll` in the pop-up modal div. but you also need to disable the `overflow-y: scroll` in the body in that case because whenever I open the pop-up modal, there is overflow-y: scroll behavior in the body. kindly consider fixing it.
<img width="489" alt="Screenshot 2023-06-28 at 9 30 00 PM" src="https://github.com/Bettenoch/Portfolio-Setup-Mobile-First/assets/96683843/3e3426a0-0f77-4461-8491-313b5bd5c1c5">